lib/manager/components/transform/FeedTransformation.js

```diff
@@ -189,6 +114,7 @@ export default class FeedTransformation extends Component<Props, {csvData: ?stri
           <Select
             style={{margin: '10px 0px', width: '230px'}}
             clearable={false}
+            searchable={false}
```

Comment:
Why make this unsearchable?

Reply:
Reverted in https://github.com/ibi-group/datatools-ui/pull/649/commits/9a4a01d4b5a48e77d70e5e34c9f20226a04a2e8e.